core/templates/pages/exploration-editor-page/editor-tab/templates/modal-templates/customize-interaction-modal.component.ts

```diff
@@ -253,11 +253,37 @@ export class CustomizeInteractionModalComponent
     this.isinteractionOpen = true;
   }
 
+  ButtonEnabledOnEndExploration(): boolean {
+    /*
+      There is a condition for the END EXPLORATION interaction
+      We have to check that the last input field for adding
+      the exploration Ids must be non-empty
+      Save Interaction button should be disable for empty input
```

Comment:
```suggestion
      For the EndExploration interaction, we check whether the last input field is empty or not.
      Since checking recommended exploration ID's validity is an expensive operation, it is only
      checked in the interaction validations and not here in real-time.
```

Reply:
done